circle.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
 test:
   override:
     - make assets
+    - make docs
```

Comment:
Since the make target `test-bdd` now depends on docs, you can actually just add it as a dependency of the said target in the Makefile:

```
test-bdd: docs # <- that one
        bin/kalite manage test --bdd-only
```